fastrag/readers/__init__.py

```diff
@@ -3,8 +3,11 @@
 from haystack.nodes import BaseGenerator, BaseReader
 from haystack.schema import Document, MultiLabel
 
-from fastrag.readers.FiD import FiDReader
-from fastrag.readers.T5 import T5Reader
+# Currently removing these classes due to incompatibility with the Seq2SeqGenerator
+# Future versions will fix this discrepancy
+
+# from fastrag.readers.FiD import FiDReader
+# from fastrag.readers.T5 import T5Reader
```

Comment:
what did you change the version? I dont see you touched `setup.cfg`

Reply:
I updated the farm-haystack version. I re-launched the demo as well with the new version to ensure it works, and it does